docs/welcome.md

```diff
@@ -57,6 +57,8 @@ You’ll then be ready to learn how to build your own features. Our tutorial for
 
 From your Unity Engine file browser, navigate to where you downloaded the FPS starter project and open the `workers/unity` directory inside the project to get started.
 
+<%(Callout type="info" message="Opening a Unity project for the first time may take while, as Unity needs to import and process the project's assets. Why not make yourself a cup of tea while you wait?")%>
```

Comment:
Do we do improbadoc syntax on master?

Reply:
this is going into the conversion branch, so I assume that is fine?

Reply:
ooooooooooooooooooooooooh.